res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1776,20 +2008,21 @@ PioneerDDJSX.hotCueLeds = function(value, group, control) {
 
 PioneerDDJSX.VuMeterLeds = function(value, group, control) {
     var midiBaseAdress = 0xB0,
-        midiOut = 0;
+        channel = 0x02,
+        midiOut = 0x00;
 
     value = 1 / (1 - PioneerDDJSX.cutVumeter) * (value - PioneerDDJSX.cutVumeter);
     if (value < 0) {
         value = 0;
     }
 
-    value = parseInt(value * 0x77); //full level indicator: 0x7F
+    value = parseInt(value * 0x76); //full level indicator: 0x7F
     if (value < 0) {
         value = 0;
     }
 
-    if (value > 0x77) {
-        value = 0x77;
+    if (value > 0x76) {
+        value = 0x76;
     }
 
     if (engine.getValue(group, "PeakIndicator")) {
```

Comment:
Okay, good. I added a new [guideline](https://mixxx.org/wiki/doku.php/contributing_mappings#level_meter_leds) to the Contributing Mappings wiki page to clarify this because when I tested the Pioneer DDJ-SB2 mapping recently I discovered the scaling of the controller's level meters did not match Mixxx, which defeated the purpose of having level meters on the controller.